### ISSUE TITLE
Package groups got renamed mysteriously 

### ISSUE THREAD

**Issue by ericwinger** [GemTalk/Rowan#792]
I don't have a reproduction case yet but noting the behavior in hopes of finding one.  Assigning to myself until a reproduction case is found but help is appreciated. 

While editing the definition for `RowanSample9` branch `spec_0061`, Somehow, several package groups got renamed with extra `groups/`: 

Some data points might help find a reproduction case: 

1.  The renamed groups had been legitimately renamed (suffixed with a number) at an earlier time. What's unexpected is the addition of the `/groups/groups/groups/` 

```
groups/groups/groups/groups/WorkGroup_dkh23
groups/groups/groups/groups/Projects23
groups/groups/Tests2
groups/groups/Examples2
```
2. One did not show the problem. It had never been renamed.  
```
groups/Rowan
```

3. I was doing package rename debugging when the problem was noticed. 
4. components don't appear to be affected.  
5. The contents (packages) of each problem package group had been earlier modified.

**Comment by ericwinger** [comment]
So the reproduction case may be simpler than I realized.  Simply renaming a package group from `groups/Examples` to `groups/Examples123` instead renamed the package group to `groups/groups/Examples`123`.

**Comment by dalehenrich** [comment]
Could you show the actual smalltalk code? ... the formatting of your english version of the code is funky so can't guess what the code actually would look like ...

**Comment by ericwinger** [comment]
<anRwDefinedProject> renamePackageGroupNamed: packageGroupName to: newPackageGroupName.

**Comment by dalehenrich** [comment]
do you have the actual original and new strings that you used?

**Comment by ericwinger** [comment]
This failed for me 
groups/Examples
to:
groups/Examples123

**Comment by dalehenrich** [comment]
Then this is the Smalltalk code:
```smalltalk
"freshly loaded RowanSample9:spec_0061"
(Rowan projectNamed: 'RowanSample9') renamePackageGroupNamed: 'groups/Examples' to: 'groups/Examples123'
```

**Comment by ericwinger** [comment]
I think you need an asDefined in there

**Comment by dalehenrich** [comment]
I think the message is #defined ... unless you WERE actually using asDefinition?

**Comment by ericwinger** [comment]
Yup, double checking code. `defined` is how I'm getting the definition.
